### ISSUE TITLE
status_code got renamed to code 

### ISSUE THREAD

**Issue by nicocesar** [nsqio/pynsq#172]
I get the following error: 

```
[I 170422 13:53:49 reader:235] [test:test] starting reader for test/test...
[E 170422 13:53:49 ioloop:633] Exception in callback functools.partial(<function wrap.<locals>.null_wrapper at 0x7f4646182598>, HTTPResponse(_body=b'{"channels":[],"producers":[{"remote_address":"172.17.0.3:36984","hostname":"648f123d078b","broadcast_address":"172.17.0.1","tcp_port":4150,"http_port":4151,"version":"1.0.0-compat"}]}',buffer=<_io.BytesIO object at 0x7f4646173a98>,code=200,effective_url='http://172.17.0.1:4161/lookup?topic=test',error=None,headers=<tornado.httputil.HTTPHeaders object at 0x7f464610f198>,reason='OK',request=<tornado.httpclient.HTTPRequest object at 0x7f464610f3c8>,request_time=0.0024971961975097656,time_info={}))
    Traceback (most recent call last):
      File "/usr/lib/python3.5/site-packages/tornado/ioloop.py", line 604, in _run_callback
        ret = callback()
      File "/usr/lib/python3.5/site-packages/tornado/stack_context.py", line 275, in null_wrapper
        return fn(*args, **kwargs)
      File "/usr/lib/python3.5/site-packages/nsq/reader.py", line 603, in _finish_query_lookupd
        if lookup_data['status_code'] != 200:
    KeyError: 'status_code'
```

**Comment by nicocesar** [comment]
I had to upgrade pynsq 0.7.1 -> 0.8.0   to make this to work. 

Code on the reader should be more robust to catch this kind of discrepancies, closing bug

**Comment by jehiah** [comment]
As you noticed this was fixed in #167 (pynsq v0.8.0) and is required for compatibility with nsqd v1.0.0-compat.
